### ISSUE TITLE
Add term fibular torsion

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10438]
**Preferred term label:**
Fibular torsion

**Synonyms**
Fibular rotation

**Definition (free text, please give PubMed ID)**
Excessive rotated (also called torsion) position of the fibula (calf bone). An increased external rotation of the fibula in the incisura tibiofibularis of more than 5° may lead to an increased pressure and will negatively affect the cartilage.
PMID: 24796832, PMID: 9409794

**Parent term (use hpo.jax.org/app)**
Abnormal fibula morphology HP:0002991

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, Acromesomelic dysplasia Grebe type (omim20070) PMID27577507,
and after ankle fracture (PMID: 9409794)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000921
- label: Fibular torsion
- parents: [Abnormal fibula morphology (HP:0002991)](https://hpo.jax.org/app/browse/term/HP:0002991)
- pmid: PMID:24796832|PMID:9409794
- def: Excessively rotated (also called torsion) position of the fibula (calf bone). An increased external rotation of the fibula in the incisura tibiofibularis of more than 5 degrees may lead to an increased pressure and will negatively affect the cartilage.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10438
- synonyms: Fibular rotation(NONE)
